TFlippy_TerritoryControl_Core_Dev/Entities/DecayIfSpammed.as

```diff
@@ -0,0 +1,36 @@
+#include "DecayCommon.as";
```

Comment:
Instead of adding a whole new base file to TC it would be wiser to just copy the code and place it in the fragmine file, since that gives more room for editing and we wont ever have to worry about file bloat & long term support. Not to mention that by editing this file you change other blobs and could potentially (probably wont) have unintended effects.

Reply:
Btw just thought of a perfect solution that wouldn't require decay at all. Simply add in collision for frag mines against other frag mines, so they cannot be stacked like before.

Reply:
Imagine reducing the lag by introducing extra collision checks